### ISSUE TITLE
Training config of wod-e2e

### ISSUE THREAD

**Issue by Debactor** [ucla-mobility/AutoVLA#47]
Hi~ Thank you for your fascinating work! 

May I please ask for the specific training configs when I sft the model with wod-e2e datasets (both with and without the CoT)?  I am planning to reproduce the results in the Table S4 row 3 and row 4. Could you please give me some suggestions?

The training config I asked may include the learning rate (lr), the total batch size, the warm-up ratio, the weight lambda in the loss and other super parametres that differ from the configs in the `[qwen2.5-vl-3B-mix-sft.yaml](https://github.com/ucla-mobility/AutoVLA/blob/main/config/training/qwen2.5-vl-3B-mix-sft.yaml)`.

**Comment by Zewei-Zhou** [comment]
Hi! Thanks for your interest in our model!

Regarding the training for the Waymo dataset (wod-e2e), we had to account for its relatively limited data volume. Here are the key adjustments we made:

We used a very short warm-up ratio, or sometimes no warm-up at all. Then, we increased the total training epochs to around 10, and we utilized a relatively larger learning rate.

As a suggestion, you might want to consider sampling more scenarios from the Waymo videos to enrich the training data volume and improve your results.
